src/components/ItemMovie/ItemMovie.test.tsx

```diff
@@ -1,29 +1,64 @@
 import ItemMovie from './ItemMovie';
-import { render, screen } from '@testing-library/react';
+import { fireEvent, render, screen, waitFor } from '@testing-library/react';
+import { Url } from '../../Api/Api';
+import { rest } from 'msw';
+import { setupServer } from 'msw/node';
 
-describe('ItemMovie', () => {
-  const movie = {
-    id: 1423589,
-    name: 'The Banshees of Inisherin',
-    countries: [
-      {
-        name: 'United Kingdom',
-      },
-      {
-        name: 'United States',
-      },
-      {
-        name: 'Ireland',
-      },
-    ],
-    year: 2022,
-    poster:
-      'https://resizing.flixster.com/SqYohHz5ela5G1qTZ8aKYunK2Oo=/ems.cHJkLWVtcy1hc3NldHMvbW92aWVzLzc4ZjI3ZDBiLTkwMDktNDVkZC1hZmI5LTJiNDdjNWRmZDVjOC5qcGc=',
-    rating: 7.7,
-  };
+const movie = {
```

Comment:
this is a very big object, better to create a folder mocks and store such value there especially if u will use them in another tests

Reply:
moved it to the mocks folder.